.github/workflows/tg-unit-tests.yaml

```diff
@@ -0,0 +1,50 @@
+name: "[TG] TG unit tests"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # This cron schedule runs the workflow every day at 12am UTC
+
+jobs:
+  build-artifact:
+    uses: ./.github/workflows/build-artifact.yaml
+    secrets: inherit
+  TG-tests:
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        runner-info: [
+          # TG
+          {arch: wormhole_b0, runs-on: ["config-tg"]},
```

Comment:
Have we thought about `in-service` or `out-of-service`? Or should we just have `in-service` and the absence of it implies `out-of-service` - per your infra cleanup proposal for labels>

Reply:
We should have in-service and out-service labels for sure, adding

Reply:
Ill starting adding it to all galaxy machines and t3ks

Reply:
👍